node/pkg/driver/node_utils.go

```diff
@@ -196,10 +177,50 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+func readFile(path string) (string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return "", err
+	}
+
+	defer file.Close()
+
+	fileOut, err := ioutil.ReadAll(file)
+	if err != nil {
+		return "", err
+	}
+
+	fileOutString := string(fileOut)
+
+	return fileOutString, nil
+}
+
+func readAfterPrefix(path string, prefix string, portType string) (string, error) {
+	fileOutString, err := readFile(path)
+	if err != nil {
+		return "", err
+	}
+
+	if prefix != "" {
+		if !strings.Contains(fileOutString, prefix) {
+			return "", fmt.Errorf(ErrorWhileTryingToReadPort, portType, fileOutString)
+		}
+		fileOutString = strings.TrimPrefix(fileOutString, prefix)
+	}
+
+	trimmedOut := strings.TrimSpace(fileOutString)
+
+	return trimmedOut, nil
+}
+
+func (n NodeUtils) ParseNvmeNqn() (string, error) {
+	return readAfterPrefix(NvmeFullPath, "", device_connectivity.ConnectionTypeNVMEoFC)
```

Comment:
```suggestion
func (n NodeUtils) ReadNvmeNqn() (string, error) {
	return readFile(NvmeFullPath)
```

Reply:
nvme need TrimSpace that is readAfterPrefix so it is parsing..

Reply:
since trim won't cause errors, could we move it to `readFile` and rename the function to something like `readTrimmedFile`?

Reply:
so why to name `parsing` any of them?

Reply:
skipping a prefix is parsing

Reply:
its trimming anyway..

Reply:
done